pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -50,7 +51,21 @@
     'mindtpy',
     doc='MindtPy: Mixed-Integer Nonlinear Decomposition Toolbox in Pyomo')
 class MindtPySolver(object):
-    """A decomposition-based MINLP solver.
+    """
+    Decomposition solver for Mixed-Integer Nonlinear Programming (MINLP) problems.
+
+    The MindtPy (Mixed-Integer Nonlinear Decomposition Toolbox in Pyomo) solver applies a variety of decomposition-based approaches to solve Mixed-Integer Nonlinear Programming (MINLP) problems. 
+    These approaches include:
+
+    - Outer approximation (OA)
+    - Global outer approximation (GOA)
+    - Regularized outer approximation (ROA)
+    - LP/NLP based branch-and-bound (LP/NLP)
+    - Global LP/NLP based branch-and-bound (GLP/NLP)
+    - Regularized LP/NLP based branch-and-bound (RLP/NLP)
+    - Feasibility pump (FP)
+
+    This solver implementation was developed by Carnegie Mellon University in the research group of Ignacio Grossmann.
```

Comment:
Rewrite as follows:
This solver implementation has been developed by David Bernal (@bernalde) and Zedong Peng (@ZedongPeng) as part of research efforts at the Grossmann Research Group (http://egon.cheme.cmu.edu/) at the Department of Chemical Engineering at Carnegie Mellon University.